ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/BlockProcessorUtil.java

```diff
@@ -238,8 +229,7 @@ public static void process_proposer_slashings(BeaconState state, BeaconBlock blo
                     DOMAIN_BEACON_BLOCK)),
             "BLSVerify fail for proposal header 2");
 
-        // - Run penalize_validator(state, proposer_slashing.proposer_index)
-        slash_validator(state, proposer_slashing.getProposer_index().intValue());
+\        slash_validator(state, proposer_slashing.getProposer_index().intValue());
```

Comment:
I think you left a random backslash here

Reply:
whoops lol thank you!